### PR TITLE
Let

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Addresses #116.

### CODE REVIEW

File: pages/Basic Types.md [hunk 1]
```diff
@@ -184,5 +187,11 @@ function warnUser(): void {
...
-```
\ No newline at end of file
+let unusable: void = undefined;
+```
+
+# A note about `let`
+
+You may've noticed that so far, we've been using the `let` keyword instead of JavaScript's `var` keyword which you might be more familiar with.
+The `let` keyword is actually a newer JavaScript construct that TypeScript makes available.
+We'll discuss the details later, but many common problems in JavaScript are alleviated by using `let`, so it's best practice to adopt it over using `var` whenever possible.
```
Comment (mhegazy) [inline pages/Basic Types.md]: except that later never happens :)

Comment (mhegazy) [inline pages/Basic Types.md]: `let` and `const` ?

**Comment by mhegazy** [comment]
:+1:

File: pages/Basic Types.md (hunk 1 above)
Comment (DanielRosenwasser) [inline pages/Basic Types.md]: That's future Daniel's problem, and he's a jerk anyhow.

**Pull request merged**